src/style.css

```
@@ -0,0 +1,110 @@
+html {
+  font-family: Roboto, sans-serif;
+}
+
+.productCard {
+  display: flex;
+  flex-direction: column;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  height: 406px;
+  width: 198px;
+}
+
+.img {
+  height: 134px;
+  width: 160px;
+  margin: 32px 19px 40px;
+  text-align: center;
+}
+
+.productCard__name {
+  text-align: left;
+  font-size: 12px;
+  font-weight: 500;
+  font-style: normal;
+  line-height: 18px;
+  margin: 0 16px;
+  margin-bottom: 4px;
+  color: #060b35;
+}
+
+.productCard__code {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+  text-align: left;
+  margin: 0 90px 0 16px;
+  margin-bottom: 16px;
+}
+
+.productCard__rating {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.stars {
+  display: flex;
+  gap: 4px;
+  margin-left: 16px;
+  list-style: none;
+}
```

тут ми повинні стилізувати сам блок зірок і зірочки всередині, а також треба задати зміну картинки для кожної stars__star. Не забудь що треба цей блок винести в окремий CSS
```suggestion
.stars {
  display: flex;
}

.stars__star {
  width: 16px;
  height: 16px;
  margin-right: 4px;
  background-image: url("../images/star.svg");
  background-repeat: no-repeat;
  background-position: center;
}
.stars--1 > :nth-child(1){
  background-image: url("../images/star-active.svg");
 } /* цю умову треба виконати відповідно для кожного з  stars--  :nth-child /*
```